README.md

```diff
@@ -209,6 +209,29 @@ This version fixes some small console spam bugs and improves the performance of
 - Updates to [QuartzLib 0.0.4](https://github.com/zDevelopers/QuartzLib/blob/master/CHANGELOG.md#quartzlib-004),
   where all these improvements were made.
 
+### 4.2.0 —The permission and cliffhanger part 1 Update
+Permission and cliffhanger Update finally allow a full official support of minecraft 1.17 and java 17. This comes with bugfixes and new features. We are working on the next big update that will be the 5.0. If you want to know more about this checkout our 5.0 milestone.
+
+This version is only compatible with Minecraft 1.15+. We have no clue if this will work with minecraft 1.18 (it should, but we are not sure)
```

Comment:
We could also consider dropping official support for 1.15?

bStats reports that only ~1000 servers are still this far behind.

Reply:
I initially planed to drop 1.15 with the 5.0.0. This should involve map format migration and lot of new things like dithering, interractive maps, "gif" map, optimization and some sort of public map player store on a server...